docs/config/server-options.md

```diff
@@ -288,6 +290,8 @@ createServer()
           searchForWorkspaceRoot(process.cwd()),
           // tus reglas personalizadas
           '/path/to/custom/allow',
```

Comment:
This line should be removed according issue.

Reply:
yes! sorry 👍